### PR TITLE
doc: Add changelog entry for BT APIs converting err to string

### PR DESCRIPTION
**Pull request by rugeGerritsen**
The new APIs will simplify debugging issues.
The APIs are used in samples and audio app, see
https://github.com/nrfconnect/sdk-nrf/pull/16729
and
https://github.com/nrfconnect/sdk-nrf/pull/16733

### CODE REVIEW

**Review by alwa-nordic — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -103,6 +103,9 @@ Bluetooth® LE
 -------------
 
 * The correct SoftDevice Controller library :kconfig:option:`CONFIG_BT_LL_SOFTDEVICE_MULTIROLE` will now be selected automatically when using coexistence based on :kconfig:option:`CONFIG_MPSL_CX` for nRF52-series devices.
+* Added the APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` to allow printing error codes as strings.
+  The APIs return string representations of the error codes when :kconfig:option:`CONFIG_BT_HCI_ERR_TO_STR` and :kconfig:option:`CONFIG_BT_SECURITY_ERR_TO_STR` are enabled.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:107]:
```suggestion
  Each API returns string representations of the error codes when the corresponding Kconfig option, :kconfig:option:`CONFIG_BT_HCI_ERR_TO_STR` or :kconfig:option:`CONFIG_BT_SECURITY_ERR_TO_STR` Kconfig, is enabled.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -103,6 +103,9 @@ Bluetooth® LE
...
+  The Bluetooth samples and :ref:`nrf53_audio_app` are updated to utilize these new APIs.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:108]:
```suggestion
  The :ref:`ble_samples` and :ref:`nrf53_audio_app` are updated to utilize these new APIs.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
The first two lines should be added to the Bluetooth samples section as well.

Maybe to the nRF5340 audio section as well. @alwa-nordic Do you think this is a notable enough change that it should be mentioned for the application section?

**Comment by wiba-nordic** [comment]
> Maybe to the nRF5340 audio section as well. @alwa-nordic Do you think this is a notable enough change that it should be mentioned for the application section?

Or I guess @alexsven who reviewed the related PR.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -212,7 +215,10 @@ Matter Bridge
 nRF5340 Audio
 -------------
 
-|no_changes_yet_note|
+* Added:
+
+  * The APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` are used to allow printing error codes as strings.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:220]:
```suggestion
  * The APIs :c:func:`bt_hci_err_to_str` and :c:func:`bt_security_err_to_str` that are used to allow printing error codes as strings.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by alexsven — APPROVED** [review]

**Pull request merged**